docs/dust_extinction/model_flavors.rst

```diff
@@ -377,7 +377,7 @@ Grain models
              ZDA04,
              C11, J13,
              HD23,
-             Y24]
+             Y24, P24]
```

Comment:
There needs to be a corresponding name in the "modelnames" variable that is right after this.  I believe this is giving the error when the docs are being built.